kmip/core/enums.py

```diff
@@ -330,10 +330,9 @@ class SecretDataType(Enum):
     PASSWORD = 0x00000001
     SEED     = 0x00000002
 
-
 # 9.1.3.2.10
 class OpaqueDataType(Enum):
-    pass
+    NONE = 0x00000000 # Not defined by the standard, but we need something
```

Comment:
Good point, something like `NONE = 0x80000000` would probably be better.
I haven't found an `OpaqueData` example in the testing documentation. Since they're non-standard, that doesn't completely surprise me but it is frustrating.